### ISSUE TITLE
Installing 4.2.0 installs the wrong version of python-http-client

### ISSUE THREAD

**Issue by w-** [sendgrid/sendgrid-python#321]
#### Issue Summary
If I install 4.2.0 via pip, it install python-http-client==2.3.0
We should be getting 2.2.1

This breaks things if we are doing error handling using HTTPError from urllib.

#### Steps to Reproduce

```
$ pip install sendgrid==4.2.0
Collecting sendgrid==4.2.0
  Using cached sendgrid-4.2.0-py2.py3-none-any.whl
Collecting python-http-client>=2.1.1 (from sendgrid==4.2.0)
  Using cached python_http_client-2.3.0-py2.py3-none-any.whl
Installing collected packages: python-http-client, sendgrid
Successfully installed python-http-client-2.3.0 sendgrid-4.2.0
```

#### Technical details:
python 2.7.13
pip 9.0.1

#### Workaround
FYI if we preinstall 2.2.1 it seems to be ok

```
$ pip install sendgrid==4.2.0
Collecting sendgrid==4.2.0
  Using cached sendgrid-4.2.0-py2.py3-none-any.whl
Requirement already satisfied: python-http-client>=2.1.1 in ./py_venvs/gfy/lib/python2.7/site-packages (from sendgrid==4.2.0)
Installing collected packages: sendgrid
Successfully installed sendgrid-4.2.0
```

**Comment by thinkingserious** [comment]
Thanks for the heads up @w-!

I've added this to our backlog for a fix.

**Comment by thinkingserious** [comment]
Fixed with version 4.2.1
